experimental/schemas/risk.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: risk
+  title: Risk score information
+  group: 2
+  short: Fields for describing the risk score and level.
```

Comment:
@ajosh0504 I stubbed this short and description since it wasn't explicit in the RFC document. Everything else should be verbatim.

Reply:
That's fine!